src/Document/User.php

```diff
@@ -108,58 +97,19 @@ public function getUserLastName()
         return $this->userLastName;
     }
     
-    /**
-     * Add an email to the collection of emails
-     *
-     * @param string $email The email to add.
-     *
-     * @return User
-     */
-    public function addEmail($email)
-    {
-        $this->emails[] = $email;
-
-        return $this;
-    }
-
-    /**
-     * Remove an email from the collection of emails
-     *
-     * @param string $email The email to disassociate from this user.
-     *
-     * @return User
-     */
-    public function removeEmail($email)
-    {
-        $this->email->removeElement($email);
-
-        return $this;
-    }
-
-    /**
-     * Get all emails in colletion
-     *
-     * @return \Doctrine\Common\Collections\Collection
-     */
-    public function getEmails()
-    {
-        return $this->emails;
-    }
-
     public function setUserEmail($userEmail)
     {
-        $this->userEmail = explode(',', $userEmail);
+        $this->userEmail = $userEmail;
     }
 
     public function getUserEmail()
     {
-        //return $this->userEmail;
-        return $this->emails;
+        return $this->userEmail;
     }
 
     public function setUserMobileNumber($userMobileNumber)
     {
-        $this->userMobileNumber = explode(',', $userMobileNumber);
+        $this->userMobileNumber = $userMobileNumber;
```

Comment:
how this will have multiple mobile numbers?

Reply:
It will allow multiple mobile numbers as its data type set as hash which will accept array values.